src/Theme/FooterTemplate.elm

```diff
@@ -0,0 +1,20 @@
+module Theme.FooterTemplate exposing (..)
+
+import Css exposing (Style, batch)
+import Html.Styled as Html exposing (Html, footer)
+import Html.Styled.Attributes exposing (css)
+
+
+type alias PageUsingTemplate msg =
+    { content : Html msg }
+
+
+view : PageUsingTemplate msg -> Html.Html msg
```

Comment:
This feels weird. Maybe let's keep it simple and just output Footer Html without passing anything in? I don't think there is much point over complicating it before we actually build